deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```
@@ -78,9 +78,9 @@ spec:
         - name: CONFIG_FILE_DST
           value: "/config/config.yaml"
         - name: DEFAULT_CONFIG
-          value: "{{ .Values.config.default }}"
+          value: {{ .Values.config.default | quote }}
```

Note that since we're dealing with a yaml file, we don't even need the quotes in the first place. Would:
```suggestion
          value: {{ .Values.config.default }}
```
not be a better cleanup?